pkg/trace/pb/tracer_payload.pb.go

```diff
@@ -0,0 +1,1216 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: tracer_payload.proto
+
+package pb
+
+import proto "github.com/gogo/protobuf/proto"
+import fmt "fmt"
+import math "math"
+
+import io "io"
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// TraceChunk represents a list of spans with the same trace id.
+type TraceChunk struct {
+	// priority specifies sampling priority of the trace.
+	Priority int32 `protobuf:"varint,1,opt,name=priority,proto3" json:"priority,omitempty"`
+	// origin specifies origin product ("lambda", "rum", etc.) of the trace.
+	Origin string `protobuf:"bytes,2,opt,name=origin,proto3" json:"origin,omitempty"`
+	// spans specifies list of containing spans.
+	Spans []*Span `protobuf:"bytes,3,rep,name=spans" json:"spans,omitempty"`
+	// tags specifies tags common in all `spans`.
+	Tags map[string]string `protobuf:"bytes,4,rep,name=tags" json:"tags,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+	// droppedTrace specifies whether the trace was dropped by samplers or not.
+	DroppedTrace bool `protobuf:"varint,5,opt,name=droppedTrace,proto3" json:"droppedTrace,omitempty"`
+}
+
+func (m *TraceChunk) Reset()                    { *m = TraceChunk{} }
+func (m *TraceChunk) String() string            { return proto.CompactTextString(m) }
+func (*TraceChunk) ProtoMessage()               {}
+func (*TraceChunk) Descriptor() ([]byte, []int) { return fileDescriptorTracerPayload, []int{0} }
+
+func (m *TraceChunk) GetSpans() []*Span {
+	if m != nil {
+		return m.Spans
+	}
+	return nil
+}
+
+func (m *TraceChunk) GetTags() map[string]string {
+	if m != nil {
+		return m.Tags
+	}
+	return nil
+}
+
+// TracerPayload represents a payload the trace agent receives from tracers.
```

Comment:
This looks awesome now! Thank you very much for adding the documentation! Now, when anyone goes to the definition of any field, this file will open up, and the documentation will clarify what it is.